docs/src/main/tut/middleware.md

```diff
@@ -102,21 +102,23 @@ newService.orNotFound(goodRequest).unsafeRunSync
 newService.orNotFound(badRequest).unsafeRunSync
 ```
 
-It is possible for the wrapped `Service` to have different `Request` and `Response`
-types than the middleware. Authentication is, again, a good example. Authentication
-middleware is an `HttpService` (an alias for `Service[Request, Response]`) that wraps an `
-AuthedService` (an alias for `Service[AuthedRequest[T], Response]`. There is a type
-defined for this in the `http4s.server` package:
+It is possible for the wrapped `Service` to have different `Request` and
+`Response` types than the `Middleware`. `Authentication` is, a good example.
+`Authentication` `Middleware` is an `HttpService` (an alias for `Service[Request,Response]`)
```

Comment:
Is it worth mentioning that `Request` and `Response` have a type parameter here?  I don't want to confuse the issue in informal descriptions, but think it might be useful here where we're showing actual signatures.

Reply:
Does the other documentation mention the type parameters? I'd vote for unification.

Reply:
I'm not sure that all the documentation has been updated for that in the areas that aren't compiled.  I think we should do it here, and then additionally take a pass through the rest.  Unify on telling it like it is.